content/chainguard/migration-guides/migrating-to-chainguard-images.md

```diff
@@ -0,0 +1,130 @@
+
+If you are coming from a Red Hat UBI based Dockerfile, you'll need to adapt some of your commands to be compatible with the apk ecosystem:
+
+| Command Description          | Red Hat UBI Dockerfile | Wolfi-based Equivalent |
+|------------------------------|------------------------|------------------------|
+| Install a package            | `yum install`          | apk add                |
+| Remove a package             | `yum remove`           | apk del                |
+| Update package manager cache | `yum makecache`        | apk update             |
+
+## Migrating from Alpine Dockerfiles
+If your Dockerfile is based on Alpine, the process for migrating to Chainguard Images should be more straightforward, since you're already using `apk` commands. Wolfi packages typically match what is available in Alpine, with some exceptions. For instance, the Wolfi busybox package is slimmer and doesn't include all tools available in Alpine's busybox. Check the [Alpine Compatibility](https://edu.chainguard.dev/chainguard/migration-guides/alpine-compatibility/) page for a list of common tools and their corresponding packages in Wolfi and Alpine.
+
+## Searching for Packages
+Packages from Debian and other base distributions might have a different name in Wolfi. To search for packages, log into an ephemeral container based on `cgr.dev/chainguard/wolfi-base`:
+
+```shell
+docker run -it --rm --entrypoint /bin/sh cgr.dev/chainguard/wolfi-base
+```
+
+Then, run `apk update` to update the local apk cache with latest Wolfi packages:
+
+```shell
+# apk update
+fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
+[https://packages.wolfi.dev/os]
+OK: 46985 distinct packages available
+```
+Now you can use `apk search` to look for packages. The following example searches for PHP 8.2 XML extensions:
+
+```shell
+# apk search php*8.2*xml*
+```
+You should get output similar to this:
+
+```shell
+php-8.2-simplexml-8.2.17-r0
+php-8.2-simplexml-config-8.2.17-r0
+php-8.2-xml-8.2.17-r0
+php-8.2-xml-config-8.2.17-r0
+php-8.2-xmlreader-8.2.17-r0
+php-8.2-xmlreader-config-8.2.17-r0
+php-8.2-xmlwriter-8.2.17-r0
+php-8.2-xmlwriter-config-8.2.17-r0
+php-simplexml-8.2.11-r1
+php-xml-8.2.11-r1
+php-xmlreader-8.2.11-r1
+php-xmlwriter-8.2.11-r1
+```
+
+### Searching which package has a command
+To search in which package you can find a command, you can use the syntax `apk search cmd:command-name`. For instance, if you want to discover which package has the command `useradd`, you can use:
+
+```shell
+# apk search cmd:useradd
+```
+You'll get output indicating that the `shadow` package has the command you are looking for.
+```shell
```

Comment:
```suggestion
```
```

Reply:
I don't think it's showing up correclty, but I don't think the `shell` is necessary ehre